src/test/java/org/ligoj/app/plugin/prov/ProvResourceTest.java

```diff
@@ -735,6 +735,32 @@ void getKey() {
 		ProvTenancy.valueOf(ProvTenancy.DEDICATED.name());
 	}
 
+	@Test
+	void getKeyDedicatedHost() {
+		Assertions.assertEquals("service:prov", resource.getKey());
+
+		// Only there for coverage of associations required by JPA
+		new ProvQuote().setStorages(null);
+		new ProvQuote().getStorages();
+		new ProvQuote().getDatabases();
+		new ProvQuote().getContainers();
+		new ProvQuote().getFunctions();
+		new ProvQuote().getTags();
+		new ProvQuote().setTags(null);
+		new ProvQuote().setSupports(null);
+		new ProvQuote().setInstances(null);
+		new ProvQuote().setDatabases(null);
+		new ProvQuote().setContainers(null);
+		new ProvQuote().setFunctions(null);
+		new ProvQuoteInstance().setStorages(null);
+		new ProvQuoteContainer().setStorages(null);
+		new UpdatedCost(0).setDeleted(null);
+		Rate.valueOf(Rate.GOOD.name());
+		ProvStorageOptimized.valueOf(ProvStorageOptimized.IOPS.name());
+		VmOs.valueOf(VmOs.LINUX.name());
+		ProvTenancy.valueOf(ProvTenancy.DEDICATED_HOST.name());
```

Comment:
You only have to add this line into the previous test. Not the whole duplicated code